packages/sanes-chrome-extension/src/extension/bsMessageHandler/externalHandler.ts

```diff
@@ -0,0 +1,62 @@
+/*global chrome*/
+import { JsonRpcErrorResponse, JsonRpcRequest } from '@iov/jsonrpc';
+import { UseOnlyJsonRpcSigningServer } from '../../logic/persona';
+
+type SigningServer = UseOnlyJsonRpcSigningServer | undefined;
+type SendResponse = (response: any) => void; // eslint-disable-line @typescript-eslint/no-explicit-any
+
+function handleWithServer(
+  signingServer: SigningServer,
```

Comment:
Type must be `UseOnlyJsonRpcSigningServer | undefined` since you don't know if the server exists when you receive an external message

Reply:
But `SigningServer` is  `UseOnlyJsonRpcSigningServer | undefined`, created this type for better readability. Am I missing something?

Reply:
Sorry, my fault. You are right. Too many type, I thought `SigningServer` is from IOV-Core. All good.